OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -40,6 +40,8 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 		public RallyPointInfo Info;
 		public string PaletteName { get; private set; }
 
+		private const uint ExtraDataSetPrimaryBuilding = 1;
```

Comment:
if (order.ExtraData == ExtraDataSetPrimaryBuilding) doesn't need comment, if (order.ExtraData == 1u) is less clear. I would keep the const, it could be changed to Enum lately when more ExtraData values are used.